Model/Reclaim.php

```diff
@@ -127,6 +136,84 @@ public function productinspector($start_id, $end_id){
         return $response;
 
     }
+
+    public function historicalcustomersubscription($start_id, $end_id, $storeId=null)
+    {
+
+        if (!$start_id || !$end_id ){ 
+            throw new NotFoundException(__('Please provide start_id and end_id'));
+        }
+
+        $storeIdFilter = $this->_storeFilter($storeId);
+
+        $subscriberCollection =$this->subscriberCollection->create()
+            ->addFieldToFilter('subscriber_id', ['gteq' => (int)$start_id])
+            ->addFieldToFilter('subscriber_id', ['lteq' => (int)$end_id])
+            ->addFieldToFilter('store_id', [$storeIdFilter => $storeId]);
```

Comment:
do we still wanna do that if $storeId is null?

Reply:
it queries for nlike storeId = null.. was basically to save some additional logic of dropping the 3rd query